src/sentry/api/endpoints/source_map_debug.py

```diff
@@ -84,27 +85,22 @@ def get(self, request: Request, project: Project, event_id: str) -> Response:
         if event is None:
             raise NotFound(detail="Event not found")
 
-        release_version = event.get_tag("sentry:release")
+        release_response = self._extract_release(event, project)
+        if type(release_response) is Response:
```

Comment:
@roggenkemper Having multiple return types like this is kind of an anti-patter. Instead what we should do is raise specific errors, catch them, and then send a response here.